### DISCUSSION TITLE
Allow fees to be declared in another currency (other than instrument currency)

### DISCUSSION THREAD

**Discussion by Sigfrodr** [ghostfolio/ghostfolio#3576] [Ideas]
Some brokers do not convert fees when trading foreign instruments.
It would be better to input the fees as they are declared by the broker (ghostfolio force to convert fees in instrument currency).

**Comment by dtslvr** [comment]
Hi @Sigfrodr

Your idea seems to be a duplicate of https://github.com/ghostfolio/ghostfolio/discussions/2666. Please join the discussion there and use the upvote functionality, thank you.
